### DISCUSSION TITLE
Update the Hunger Games link in the Robotoff notification in slack to lead contributors to the most relevant logo view

### DISCUSSION THREAD

**Discussion by teolemon** [openfoodfacts/robotoff#1231] [Ideas]
### Problem
Update the Hunger Games link in the Robotoff notification in slack to lead contributors to the most relevant logo view
<img width="434" alt="image" src="https://user-images.githubusercontent.com/1689815/202495083-22feed41-8e1f-47f0-911c-dd3026ce39ff.png">

### Part of
- #424

**Comment by raphael0202** [comment]
Can you describe the issue more precisely? What has to be done?

**Comment by teolemon** [comment]
I believe I was thinking of the new logo views by @alexfauquette 
But I have a hard time choosing which one is the best/right one to use

**Comment by alexfauquette** [comment]
If you want to annotate logos similar to a specific logo, there is only one view, the historical one:
https://hunger.openfoodfacts.org/logos?logo_id=173011&count=50

The two others are based on annotation value (for example you need to provide `brand="cora"`)
